dataset/dictionary/quests.json5

```
@@ -91,21 +91,25 @@
   },
 
   // Inazuma
+  // Note: missing chapter 2 Prologue (zhCN: "振袖秋风问红叶",)
```

Oops, I didn't add it.

```suggestion
  {
    en: "Autumn Winds, Scarlet Leaves",
    ja: "秋風に舞いし紅葉",
    zhCN: "振袖秋风问红叶",
    pronunciationJa: "あきかぜにまいしもみじ",
    tags: [ "inazuma", "quest-archon" ],
  },
```